internal/gengapic/client_init_test.go

```diff
@@ -21,12 +21,12 @@ import (
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/protoc-gen-go/descriptor"
 	"github.com/golang/protobuf/ptypes/duration"
+	wrappers "github.com/golang/protobuf/ptypes/wrappers"
```

Comment:
why the reordering? what's the sort criteria?

Reply:
That's automatic changes by running `go fmt`